examples/terraform/azure/variables.tf

```diff
@@ -69,13 +69,13 @@ variable "location" {
 
 variable "control_plane_vm_size" {
   description = "VM Size for control plane machines"
-  default     = "Standard_B2s"
+  default     = "Standard_F2"
   type        = string
 }
 
 variable "worker_vm_size" {
   description = "VM Size for worker machines"
-  default     = "Standard_B2s"
+  default     = "Standard_F2"
```

Comment:
Is this going to cause the VMs to be recreated or only updated in-place? Regardless, I'd mention this in the release notes.

Reply:
That will force-replace VMs yes. However during pre-release testing `Standard_F2` proved to be much faster. Also `Standard_F2` is used in MC e2e tests and our account has wide-big quota for `Standard_F2`s.

Reply:
Can you add a note about this in the release notes? This sounds like a breaking change, so users should be aware of this.

Reply:
✅